care/utils/tests/test_base.py

```diff
@@ -0,0 +1,159 @@
+import abc
+import datetime
+
+from django.contrib.gis.geos import Point
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from care.facility.models import Facility, LocalBody, User
+from care.users.models import District, State
+
+
+class TestBase(APITestCase):
+    """
+    Base class for tests, handles most of the test setup and tools for setting up data
+    """
+
+    @classmethod
+    def create_user(cls, district: District, username: str = "user", **kwargs):
+        username = kwargs.get("username", username)
```

Comment:
`username` will either be passed as a keyword-based argument or not. Should we expect both?

Reply:
username will have a default value of "user". If nothing is passed, this default will be used. If it is passed, then the passed value will be used.